web/src/lib/components/Profile.svelte

```diff
@@ -124,8 +137,8 @@
 <input type="checkbox" id="profile-modal-toggle" for="profile-modal" class="modal-toggle" />
 <div class="modal">
     <div class="modal-box relative flex justify-center items-center w-10/12 max-w-1xl">
-        {#if $user && $user.twitterUsername && (!$user.hasAuctions || $user.contributionPercent !== null)}
-            <label for="profile-modal" class="btn btn-sm btn-circle absolute right-2 top-2" on:click={hide}>✕</label>
+        {#if $user && !$user.hasAuctions}
```

Comment:
So what happens now if the user has no auctions? Will the label not show up? Why do we need this check here at all? I thought the point was to show the label in every case, ad just take different actions depending on the case.